### ISSUE TITLE
'schema' attribute not written

### ISSUE THREAD

**Issue by rzezeski** [basho/yokozuna#129]
Occasionally the `<core>` attribute `schema` is lost or not properly written in `solr.xml`.  I'm not sure the cause.  I believe it is a bug in Solr but need to investigate.  I've personally seen this once myself and user on IRC reported the same issue.

E.g. here is what the core entry looks like:

```
<core loadOnStartup="true" instanceDir="/opt/yokozuna/riak/rel/riak/data/yz/my_core/" transient="false" name="my_core" collection="my_core"/>
```

But it should look like this (substitue `_yz_default.xml` for the schema name if a custom one is used):

```
<core schema="_yz_default.xml" loadOnStartup="true" instanceDir="/opt/yokozuna/riak/rel/riak/data/yz/my_core/" transient="false" name="my_core" collection="my_core"/>
```

I think this has something to do with the recent solr.xml changes.

http://wiki.apache.org/solr/CoreAdmin
http://wiki.apache.org/solr/Solr.xml%204.4%20and%20beyond

E.g. I don't think the `collection` attribute was there before.  I hope SolrCloud is not becoming a required feature to run Solr.

**Comment by rzezeski** [comment]
The following Solr issue may be related which was fixed in 4.4.

https://issues.apache.org/jira/browse/SOLR-4910
https://issues.apache.org/jira/browse/SOLR-4862

**Comment by rzezeski** [comment]
This should be fixed with the upgrade to 4.4.0.
